Libs/Image/Image.cpp

```diff
@@ -14,6 +14,8 @@
 #include <itkImageSeriesReader.h>
 #include <itkGDCMImageIO.h>
 #include <itkGDCMSeriesFileNames.h>
+#include "itkThinPlateSplineKernelTransform.h"
```

Comment:
actual ITK includes should use braces, not quotes (I think these aren't the versions in our tree)